src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -132,7 +135,16 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
             string scriptContent;
             using (var stopwatch = logger.LogTimedEvent("GenerateBuildScript"))
             {
-                var scriptGenerator = new BuildScriptGenerator(console, serviceProvider);
+                var checkerMessages = new List<ICheckerMessage>();
+                var scriptGenerator = new BuildScriptGenerator(serviceProvider, console, checkerMessages);
+
+                if (checkerMessages.Count > 0)
+                {
+                    var messageFormatter = new DefinitionListFormatter();
+                    checkerMessages.Select(msg => messageFormatter.AddDefinition(msg.Level.ToString(), msg.Content));
+                    console.WriteLine(messageFormatter.ToString());
```

Comment:
`console.Error.WriteLine`?

Reply:
Should we have a header or something which shows that these warnings came from Oryx?